packages/google_mlkit_image_labeling/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: google_mlkit_image_labeling
 description: A Flutter plugin to use Google's ML Kit Image Labeling to detect and extract information about entities in an image across a broad group of categories.
-version: 0.3.0
+version: 1.0.0
```

Comment:
change version to 0.4.0, we are not releasing 1.0.0 yet

Reply:
Done in 72c7a9eeb9f867ae4ecd3733f41e635b8441e3fc and 83d8a5c5fb26e5a9ed2d7f99d5ef0b667c211e9f